<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpectrumDataServer</class>
 <widget class="QWidget" name="SpectrumDataServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpectrumDataServer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>数据推送服务</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1" rowspan="2">
       <widget class="QLineEdit" name="lineEdit_ip">
        <property name="styleSheet">
         <string notr="true">height: 40px;
font-size:20px;</string>
        </property>
        <property name="text">
         <string>0.0.0.0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" rowspan="2">
       <widget class="QSpinBox" name="spinBox_port">
        <property name="focusPolicy">
         <enum>Qt::WheelFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">height: 40px;
font-size:20px;</string>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="keyboardTracking">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
        <property name="value">
         <number>12345</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="6">
       <widget class="QTextEdit" name="textEdit"/>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font-size:20px;</string>
        </property>
        <property name="text">
         <string>端口号</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5" rowspan="2">
       <widget class="QPushButton" name="btn_listen">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 85, 255);
height: 40px;
font-size:20px;</string>
        </property>
        <property name="text">
         <string>开始监听</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font-size:20px;</string>
        </property>
        <property name="text">
         <string>IP地址</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" rowspan="2">
       <widget class="QPushButton" name="btn_settings">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 85, 255);
height: 40px;
font-size:20px;</string>
        </property>
        <property name="text">
         <string>参数设置</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
